### DISCUSSION TITLE
Fracture propagation in natural fractured reservoir

### DISCUSSION THREAD

**Discussion by miraboreasumr** [idaholab/moose#23017] [Q&A General] [answered]
Hello,

I have a naturally fractured reservoir, and I have the location of the natural fractures (EDFM?)
What are the modules I should learn if I want to study the natural fracture propagation while injecting fluid into the fractures?

**Comment by jiangwen84** [comment, marked as answer]
For reservoir modeling, you want to look at porous_flow ( https://mooseframework.inl.gov/modules/porous_flow/index.html ). They have a few examples of flow through fractures, see https://mooseframework.inl.gov/modules/porous_flow/flow_through_fractured_media.html.

https://mooseframework.inl.gov/modules/porous_flow/multiapp_fracture_flow_PorousFlow_3D.html might be closest approach to the EDFM approach you mentioned. However, I do not think that is capable of modeling fracture propagation yet.

**Reply by miraboreasumr** [reply]
So no way to do fracture propagation?

**Reply by jiangwen84** [reply]
Which methods do you want to use for fracture propagation? Does EDFM support crack propagation? I am working on the implementation of a phase-field fracture models for hydraulic fracturing within porous flow module. Is that something you are interested in? I might be able to have it ready in January.

**Reply by miraboreasumr** [reply]
Thanks for the reply. EDFM+XFEM (Extended finite element method) can do fracture propagation. If your phase-field models can handle when there are natural fractures (EDFM or inner boundary?), and propagation, I will be interested in it.
